docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+* Take advantage of [Chromium's built-in emulation mode](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#use-chromium-device-emulation)
+
+For the sake of this tutorial, let's pick the third option, as it offers greater stability and speed.
+
+First, let's write a simple test that checks if the 'Add to Cart' button exists:
+
+```js
+test('`Add To Cart` button should exist', async t => {
+    await t
+        .click(Selector('button').withText('Add To Cart'));
+});
+```
+
+Now let's start automating.
+
+## Basic setup
+
+### Step 1. Take care of the prerequisites
+
+TestCafe requires a working Node.js installation to run. If your testing environment does not contain node.js, [this Jenkins plugin](https://plugins.jenkins.io/nodejs/) can install it for you. Make sure you have Chromium ready to go, too: TestCafe cannot function without a browser.
+
+### Step 2. Install the TestCafe Jenkins plugin
+
+To install the TestCafe Jenkins plugin, press the "Manage Plugins" link on the Manage Jenkins page, select the "Available" tab, and enter "testcafe" into the search field. Check the box next to our plugin and press the 'Install without restart' button below it.
+
+![Jenkins plugin search UI](/testcafe/images/jenkins-plugin/plugin-search-ui.png)
+
+### Step 3. Install the necessary Node packages
+
+Install the main testcafe library, as well as the [testcafe-reporter-jenkins](https://www.npmjs.com/package/testcafe-reporter-jenkins) package. The latter is needed to generate Jenkins-friendly JUnit format reports.
+
+To do this, add the following *build step* to your Jenkins project:
+
+```bash
+npm i testcafe testcafe-reporter-jenkins
+```
+
+### Step 4. Configure the tests
+
+You can configure your tests via:
+
+* the [command line interface](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html)
+* the JavaScript/TypeScript API
+* the .testcaferc.json [configuration file](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html)
+
+The last option provides an easy, declarative way to define your test settings. You can store this file in the same folder as your tests.  Below is the configuration we're going to use in this tutorial:
+
+```js
+{
+    "browsers": [
+        "chromium:headless:emulation:device=iPhone 4",
+        "chromium:headless:emulation:device=iPad Mini",
+        "chromium:headless"
+    ],
+    "src": "test/e2e/**/*",
+    "screenshots": {
+        "takeOnFails": true,
+        "fullPage": true
+    },
+    "videoPath": "videos",
+    "reporter": {
+        "name": "jenkins",
+        "output": "report.xml"
+    },
+    "appCommand": "node server.js"
+}
+```
+
+The *Browser* array contains the list of browsers TestCafe will use to run our test. In it, we specified the mobile devices we want Chromium to emulate.
```

Comment:
```suggestion
The *Browser* array contains the list of browsers TestCafe will use to run our test. The example above specifies the mobile devices we want Chromium to emulate.
```

Reply:
Suggestion:
``` markdown 
The Browser array contains the list of browsers TestCafe will use to run our test. The [[TestCafe browser syntax](link to docs)] helped us specify the mobile devices we want Chromium to emulate.
```

Rationale: The reader might want to know how exactly these devices were specified